Assets/Scripts/Slice.cs

```diff
@@ -0,0 +1,92 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.SceneManagement;
+
+public class Slice : MonoBehaviour
+{
+
+
+    private GameObject doll;
+    private GhostSwitchManager manager;
+    private Animator anim;
+
+    // Use this for initialization
+    void Start()
+    {
+        manager = GameObject.FindGameObjectWithTag("LevelManager").GetComponent<GhostSwitchManager>();
+        anim = GetComponent<Animator>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+    }
+
+    void OnTriggerStay2D(Collider2D other)
+    {
+        doll = other.gameObject;
+        if (/*doll.gameObject.CompareTag("Player")*/ other.tag == "Player")
+        {
+            doll.GetComponent<Controller2>().enabled = false;
+            StartCoroutine("SoulSlice");
+        }
+
+        else if ( /*doll.gameObject.CompareTag("Doll")*/ other.tag == "Doll")
+        {
+
+            doll.GetComponent<Controller2>().enabled = false;
```

Comment:
That is because in the few seconds when the animation plays the player can control the to still, so I want them to not be able to control the doll when the animation plays..cuz its getting sliced